File: 5kyu/Human Readable Time/time.py
# Write a function, which takes a non-negative integer (seconds) as input and returns the time in a human-readable format (HH:MM:SS).

def make_readable(seconds):
    hour = seconds // 3600
    minutes = seconds // 60
    minutes %= 60
    sec = seconds % (24 * 3600)
    sec %= 3600
    sec %= 60
    return "%02d:%02d:%02d" % (hour, minutes, sec)


# make_readable(359999)
# make_readable(86399)
# make_readable(60)
# make_readable(5)
# make_readable(0)

File: 5kyu/Human Readable Time/time.php
<?php
// Write a function, which takes a non-negative integer (seconds) as input and returns the time in a human-readable format (HH:MM:SS).

function human_readable($seconds) {
    $hours = floor($seconds / 3600);
    $minutes = floor(($seconds / 60) % 60);
    $seconds = $seconds % 60;
  
    return sprintf("%02d:%02d:%02d", $hours, $minutes, $seconds);
}

// human_readable(359999);
// human_readable(86399);
// human_readable(60);
// human_readable(5);
// human_readable(0);